normandy/recipes/static/actions/feature-recommendation/index.js

```diff
@@ -0,0 +1,23 @@
+import {Action, registerAction} from '../utils';
+
+export default class FeatureRecommendationAction extends Action {
+    async execute() {
+        var recCount = this.recipe.arguments.recommendations.length;
+        var domain = this.recipe.arguments.domain;
+        var recs = [];
+        for(var i = 0; i < recCount; i++) {
+          var rec = this.recipe.arguments.recommendations[i];
+          recs[i] = {
+            name: rec.name,
+            id: rec.id,
+            description: rec.description,
+            packageURL: rec.packageURL,
+            imageURL: rec.imageURL,
+            infoURL: rec.infoURL,
+          };
```

Comment:
On the other hand, it seems like you are just duplicating `this.recipe.arguments.recommendations` into `recs`. Why not call `this.normandy.recommendFeature` with `this.recipe.arguments.recommendations` directly?